src/components/Subtitle.jsx

```diff
@@ -1,5 +1,16 @@
-const Subtitle = ({ title }) => {
-  return <div>Subtitle</div>;
+import React from "react";
+const Subtitle = ({ title = "| Motivation" }) => {
+  // const Subtitle = ({ title }) => {
+  const blueColor = "#1151D3";
+  const greenColor = "#80C393";
+  return (
+    <div>
+      <h2 style={{ color: blueColor }}>{title}</h2>
```

Comment:
use p tag
add left border: border-l-4 border-ewb-green
use tailwind to color the text: text-ewb-blue-200
give a text size